test/gleam/string_test.gleam

```diff
@@ -1392,3 +1392,21 @@ pub fn inspect_map_test() {
   |> string.inspect
   |> should.equal("dict.from_list([#(\"a\", 1), #(\"b\", 2)])")
 }
+
+pub fn strip_prefix_test() {
+  let assert Ok("gleam.run") =
+    string.strip_prefix("https://gleam.run", "https://")
+  let assert Ok("https://gleam.run") =
+    string.strip_prefix("https://gleam.run", "")
+  let assert Ok("") = string.strip_prefix("", "")
+  let assert Error(Nil) = string.strip_prefix("https://gleam.run", "Lucy")
+  let assert Error(Nil) = string.strip_prefix("", "Lucy")
```

Comment:
can you please also add a test for the case I described, just to document it? :pray: